docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+# The environment variable TILED_SINGLE_USER_API_KEY must be set, as in:
+#
+#     TILED_SINGLE_USER_API_KEY=secret docker-compose up
+#
+# or using a environment file as described in the docker-compose documentation:
+#
+#     https://docs.docker.com/compose/environment-variables/set-environment-variables/
+#
+# Test using, for example:
+#
+#     python examples/write_and_read_back_data.py
+
+version: "3.2"
+services:
+  tiled:
+    image: ghcr.io/bluesky/databroker:v2.0.0b14
+    volumes:
+      - type: bind
+        source: ./example_config.yml
+        target: /deploy/config.yml
+    environment:
+      - TILED_SINGLE_USER_API_KEY=$TILED_SINGLE_USER_API_KEY
+    ports:
+      - 8000:8000
+  mongo:
+    image: docker.io/mongo:6.0.4
+    ports:
+      - 27017:27017
```

Comment:
This is a little picky, but...some IT departments will ding you for exposing a database port. How hard would it be to add a third service, based on python, for the example code that contains bluesky dependencies? Then it could mount the example file, and one could run it with `podman-compose exec example-container python write_and_read_back_data.py`. Then that port wouldn't exposed AND the user could run without building their own env/conda environment.

Too much?

Reply:
bluesky-pods has exactly this (but with terrible bash scripts).

Might be simpler to also stuff a minimal kafka cluster into the pod and expose that as a write end poin.

Reply:
This occurred to me to. As databroker already exposes a `GET /documents/{path}` endpoint, the simplest thing may be to add a `POST /documents/{path}` endpoint.

For production deployments, Kafka is the way to go, but for the simplest sandbox, sticking to HTTP would be a nice option to have. This would require some databroker development work though, so I had in mind doing this in a separate PR.

Reply:
As of commit 3856ac7, the database is not exposed:

```
telnet 127.0.0.1 27017
Trying 127.0.0.1...
telnet: Unable to connect to remote host: Connection refused
```

Reply:
That's very cool.